### PR TITLE
.

### PR DESCRIPTION
**Pull request by arshakpoghosyan**
adding login form

### CODE REVIEW

File: css/header.css [hunk 1]
```diff
@@ -109,3 +112,67 @@
...
+    outline: none;
+    border: 1.5px solid #4267B2;
+    border-radius: 5px;
+    text-indent: 5px;
+}
+
+.login_button {
+    background-color:#c48600;
+    border-radius: 6px;
+    color: white;
```
Comment (mikayel00) [inline css/header.css]: please write with hex or rgb

File: ticket.html [hunk 2]
```diff
@@ -42,7 +42,24 @@
...
+                                     </div>
+                                     <div class="login_input">
+                                         <input type="password" placeholder="Password">
+                                     </div>
+                                     <a href="#" class="forgot_pass">Forgotten password?</a>
+                                     <div>
+                                         <input type="button" class="login_button" value="Log in">
+                                     </div>
+                                 </form>
+                             </div>
```
Comment (mikayel00) [inline ticket.html]: lost `</div>`

**Review by mikayel00 — CHANGES_REQUESTED** [review]

**Review by mikayel00 — APPROVED** [review]

File: css/header.css [hunk 3]
```diff
@@ -95,6 +95,9 @@
     .menu__item {
         margin: 6px;
     }
+    .login_container {
+        right: 10px;
+        }
```
Comment (lilithkarapetyan) [inline css/header.css]: move the closing } right under the .

File: css/header.css [hunk 4]
```diff
@@ -109,3 +112,66 @@
...
 
 }
+  
+
+.login:hover .login_container{
+    display: block;
+}
+.login_container{
+    background-color: #ffffff;
+    padding: 25px;
```
Comment (lilithkarapetyan) [inline css/header.css]: if possible, please use even pixels like 24px or 26px

File: css/header.css [hunk 5]
```diff
@@ -109,3 +112,66 @@
...
+    border-radius: 6px;
+    height: 220px;
+    width: 200px;
+    margin-top: 0px;
```
Comment (lilithkarapetyan) [inline css/header.css]:
```suggestion
    margin-top: 0;
```

File: css/header.css [hunk 6]
```diff
@@ -109,3 +112,66 @@
...
+    display: none;
+    position: absolute;
+    z-index: 1;
+    align-items: center;
+    text-align: center;
+    right: 15px;
+}
+.login_container h2 {
+        color: #c48600;
```
Comment (lilithkarapetyan) [inline css/header.css]:
```suggestion
    color: #c48600;
```

File: css/header.css [hunk 7]
```diff
@@ -109,3 +112,66 @@
...
+.login_button {
+    background-color:#c48600;
+    border-radius: 6px;
+    color: rgb(255, 255, 255);
+    padding: 8px 50px;
+    text-align: center;
+    font-size: 16px;
+    border: none;
+    cursor: pointer;
+  }
```
Comment (lilithkarapetyan) [inline css/header.css]:
```suggestion
}
```

File: css/header.css [hunk 8]
```diff
@@ -109,3 +112,66 @@
...
+
+  ::placeholder{
```
Comment (lilithkarapetyan) [inline css/header.css:162]: are you sure this is working?

File: css/header.css [hunk 9]
```diff
@@ -109,3 +112,66 @@
...
+      padding-left: 7px;
+  }
+  .forgot_pass{
+      color: rgb(0, 0, 0);
```
Comment (lilithkarapetyan) [inline css/header.css]:
```suggestion
      color: #000000;
```

File: css/header.css [hunk 10]
```diff
@@ -109,3 +112,66 @@
...
+      font-size: 12px;
+      float: none;
+      color: rgb(0, 0, 0);
```
Comment (lilithkarapetyan) [inline css/header.css]:
```suggestion
      color: #000000;
```

**Review by lilithkarapetyan — APPROVED** [review]

**Pull request merged**